hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -72,6 +72,8 @@ def normalize_dict_for_df(dict_old: Dict[ResultsKey, Any]) -> Dict[str, Any]:
                 value = value.squeeze(0).cpu().numpy()
                 if value.ndim == 0:
                     value = np.full(bag_size, fill_value=value)
+            if isinstance(value, List) and isinstance(value[0], torch.Tensor):
```

Comment:
Can you explain why do we need to do that here? is that for the coordinates? Maybe turn it into numpy arrays in the first place?